starters/cra-rxjs-styled-components/src/components/Loader/Loader.test.tsx

```diff
@@ -0,0 +1,10 @@
+import { render, screen, fireEvent } from '@testing-library/react';
+import { Loader } from './Loader';
+import { MemoryRouter } from 'react-router-dom';
+
+describe('Add Loader', () => {
+  it('should appear while waiting for response', () => {
+    const { container } = render(<Loader />, { wrapper: MemoryRouter });
```

Comment:
It's recommended to use `screen` instead in the newer versions of `@testing-library` by its author Kent C Dodds
>The benefit of using screen is you no longer need to keep the render call destructure up-to-date as you add/remove the queries you need. You only need to type screen. and let your editor's magic autocomplete take care of the rest.

> The only exception to this is if you're setting the container or baseElement which you probably should avoid doing (I honestly can't think of a legitimate use case for those options anymore and they only exist for historical reasons at this point).